apache/incubator-zeppelin
# Improve/split paragraph html

### What is this PR for?
This PR is the first step to divide the paragraph into smaller components.
In order to work gradually towards making paragraph.js more maintainable, this PR will focus only on splitting the HTML.

### What type of PR is it?
Refactoring

### Todos
* [x] - Split Pragraph HTML
* [x] - Split Notebook action bar HTML
* [x] - Fix some z-index problems
* [x] - Fix failing selenium test

### Is there a relevant Jira issue?
No

### How should this be tested?
* There wasn't any real code changes, just moving code, so you can just use Zeppelin Normally
* For the z-index fixes, you can check that the paragraph menu, progress bar and graph tooltip are not showing on top of the navbar and actionbar

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
